pom.xml

```diff
@@ -2249,7 +2248,7 @@
                 <param>*.Input*</param>
               </excludedTestClasses>
               <coverageThreshold>100</coverageThreshold>
-              <mutationThreshold>100</mutationThreshold>
+              <mutationThreshold>97</mutationThreshold>
```

Comment:
If we reduce it, it might cause leak in other mutators

Reply:
It is being reduced to 99%. List is being created of outstanding items.

Reply:
while we have validation in shell script we are safe.